fenix/app/nimbus.fml.yaml

```diff
@@ -212,14 +212,6 @@ features:
         type: Boolean
         default: true
 
-  extensions-process:
-    description: A feature to rollout the extensions process.
-    variables:
-      enabled:
-        description: If true, the extensions process is enabled.
-        type: Boolean
-        default: true
-
```

Comment:
Normally for features we don't remove the nimbus flag, we only make them default to `true`, so we always have the avenue to experiment with them. Is there any particular reason why defaulting to `true`, is not sufficient?

Reply:
We don't need the kill switch anymore.